brian2tools/mdexport/mdexporter.py

```diff
@@ -34,6 +35,10 @@ def build(self, direct_call=True, debug=False, expander=None,
             If mentioned, the markdown text would be written in that
             particular filename. When empty string '' is passed the user file
             name would be taken
+
+
+        format : array
+                Array of string should be given as a input for this.    
```

Comment:
This needs to list the possible options. Also, it is not an `array`, but a `list` (or more general `sequence`/`iterable`). As I mention below, it would be good if it could accept a single string as well.